lib/project/cordova-project.ts

```diff
@@ -78,7 +78,8 @@ export class CordovaProject extends frameworkProjectBaseLib.FrameworkProjectBase
 	}
 
 	public adjustBuildProperties(buildProperties: any, projectData?: IProjectData): any {
-		buildProperties.CorePlugins = projectData.CorePlugins || [];
+		var configurationName = (options.release || options.r) ? "release" : "debug";
```

Comment:
In fact yes, in master branch we can use only options.release now (if my fix is working as expected)